files/eventbridge-ingressroute-gateway.yaml

```diff
@@ -36,4 +36,21 @@ spec:
     tls:
       minimumProtocolVersion: "1.2"
       secretName: ##CERT_NAME##
-status: {}
\ No newline at end of file
+  includes:
+  - name: cadvisor
+    namespace: vmware-system
+---
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  labels:
+    app: cadvisor
+  name: cadvisor
+  namespace: vmware-system
+spec:
+  routes:
+  - conditions:
+    - prefix: /top
+    services:
+    - name: cadvisor
+      port: 31194
```

Comment:
Is there a reason for this specific port? Looks like it's getting into `nodePort` range.